### DISCUSSION TITLE
The account management menu is not displayed

### DISCUSSION THREAD

**Discussion by mollihan** [JustArchiNET/ASF-ui#1686] [Q&A]
I decided to log in to steam from a new device. I log into ASF, select an authorized account for the 2fa code and see this:
(there should be an account menu. Activation of 2fa keys, etc.)

![image](https://github.com/user-attachments/assets/1f1a3532-04f1-4504-92b8-08fc43b01087)

Rebooting didn't help. Plugins:
It's the same problem with them and without them

![image](https://github.com/user-attachments/assets/416c2188-5d41-48e1-94c5-08ec34c083c9)

can anyone help with this?

**Comment by JustArchi** [comment]
Not related to ASF, moving to ASF-ui repo.
